cloud/gcp/modules/compute/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "region" {
+  type        = string
+  description = "Default region for the application"
+}
+
+variable "db_connection_name" {
+  type        = string
+  description = "DB connection string"
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "Name of the file storage bucket"
+}
+
+variable "http_port" {
+  type        = string
+  description = "Port to open on container for HTTP requests"
+}
+
+variable "application_service_account_email" {
+  type        = string
+  description = "The service account application runs with"
+}
+
+variable "db_secret_id" {
+  type        = string
+  description = "Secret Manager secret id for database password"
+}
+
+variable "civiform_image_name" {
```

Comment:
can you clarify, maybe in the comment what this is? is that the tag/version? e.g. prod/latest/snapshot?

Reply:
updated the description